met-web/src/services/surveyService/index.ts

```diff
@@ -1,8 +1,57 @@
 import http from 'apiManager/httpRequestHandler';
 import { Survey } from 'models/survey';
 import Endpoints from 'apiManager/endpoints';
+import { replaceUrl } from 'helper';
 
 export const fetchAllSurveys = async (): Promise<Survey[]> => {
     const responseData = await http.GetRequest<Survey[]>(Endpoints.Survey.GET_ALL);
     return responseData.data.result ?? [];
 };
+
+export const getSurvey = async (surveyId: number): Promise<Survey> => {
+    const url = replaceUrl(Endpoints.Survey.GET, '<survey_id>', String(surveyId));
+    if (!surveyId || isNaN(Number(surveyId))) {
+        return Promise.reject('Invalid Survey Id ' + surveyId);
+    }
+    try {
+        const response = await http.GetRequest<Survey>(url);
+        if (response.data.result) {
+            return Promise.resolve(response.data.result);
+        }
+        return Promise.reject(response.data.message ?? 'Failed to fetch survey');
```

Comment:
talking about constants, I think its worthwhile in the future we have all messages centralized in one file, not a scope of this PR!